hudi-cli/src/main/java/org/apache/hudi/cli/commands/MetadataCommand.java

```diff
@@ -40,25 +45,32 @@
 
 import java.io.FileNotFoundException;
 import java.io.IOException;
+import java.util.ArrayList;
 import java.util.Arrays;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.HashMap;
+import java.util.HashSet;
 import java.util.List;
 import java.util.Map;
+import java.util.Set;
 
 /**
  * CLI commands to operate on the Metadata Table.
```

Comment:
Can you add an example here. 
for eg:
connect
set spark env variables
sample metadata command.

Reply:
Added the java doc